### PR TITLE
virt-chroot tap debug actions and information additions

### PR DESCRIPTION
**Pull request by EdDev**
**What this PR does / why we need it**:
Add debug information to the virt-chroot binary when creating a tap device and use it from network phase 1.

The following actions/information are added with this change:
- Allow to retry the tap device creation.
- Allow to create a dummy-bridge as a ref to the tap creation attempt (to check if both or only one fails).
- Report the network links seen by the process on failure to create the tap device.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
This PR depends on #5388 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by EdDev** [comment]
Use for debugging at the moment.
We should reconsider this only if we fail to recreate the issue.

/hold

**Review by qinqon — CHANGES_REQUESTED** [review]
We can use rtry package

File: cmd/virt-chroot/tap-device-maker.go [hunk 1]
```diff
@@ -65,11 +75,33 @@ func NewCreateTapCommand() *cobra.Command {
...
+			fmt.Printf("Operation succeeded [%d]\n", attemptID)
+			break
+		}
+	}
+	if len(errorsString) > 0 {
+		return fmt.Errorf(strings.Join(errorsString, "\n"))
+	}
+
+	return nil
+}
```
[the remark this answers is not in the document]
Comment (EdDev) [reply cmd/virt-chroot/tap-device-maker.go:143]: Yes we probably can, but I really do not want to add dependencies on something that simple.
We also use the the cobra-command and fetch the flag that is currently specific to the tap-create command.

This is also not targeted to get merged, just to provide debug info.

**Comment by EdDev** [comment]
Incident occurred:  https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/5390/pull-kubevirt-e2e-k8s-1.18/1379189784194322432

**Pull request closed without merging**